evil-owl.el

```diff
@@ -155,17 +155,23 @@ show, and outputs an entry string (newline included)."
    groups
    evil-owl-separator))
 
+(defun evil-owl--trim-contents (contents)
+  "Return a string no more than `evil-owl-register-char-limit' characters in length."
+  (replace-regexp-in-string
+   "\n" "^J"
```

Comment:
I think we should leave this newline replacement in `evil-owl--get-register`. For marks, if we're using single-line contexts and there's a newline, it'll always be the last character. I would just strip the newline in `evil-owl--get-mark`.

Reply:
Yeah, sorry about that; I realized this didn't make sense just after I initially submitted.